core/src/main/java/com/ugamsol/core/models/Imple/HomeAboutImpl.java

```diff
@@ -1,18 +1,32 @@
 package com.ugamsol.core.models.Imple;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
 import com.ugamsol.core.models.HomeAbout;
 import org.apache.sling.api.resource.Resource;
 import org.apache.sling.models.annotations.DefaultInjectionStrategy;
+import org.apache.sling.models.annotations.Exporter;
+import org.apache.sling.models.annotations.ExporterOption;
 import org.apache.sling.models.annotations.Model;
 
 import javax.inject.Inject;
 
 @Model(adaptables = Resource.class,
         adapters = HomeAbout.class,
+        resourceType = HomeAboutImpl.RESOURCE_TYPE,
         defaultInjectionStrategy = DefaultInjectionStrategy.OPTIONAL)
+
+@Exporter(name="jackson", extensions = "json", selector = "mydemo",
+options = {
+        @ExporterOption(name = "SerializationFeature.WRAP_ROOT_VALUE", value ="true")
+}
+)
+@JsonRootName("NEW ROOT NAME")
 public class HomeAboutImpl implements HomeAbout {
 
 
+    public static final String RESOURCE_TYPE = "/apps/ugamsol/components/content/homeabout";
```

Comment:
use ugamsol/components/content/homeabout instead of full path.
same with other classes

Reply:
I have removed the /apps from all the 3 classes.